src/module-1/card/style.css

```diff
@@ -0,0 +1,100 @@
+html {
+    background-color: whitesmoke;
+}
+
+
+.card{
+    width: 400px;
+    font-family: 'Montserrat', sans-serif;
+    background-color: white;
+    border-radius: 8px;
+}
+
+.card__rating{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.card-rating__stars{
+    height: 30px;
+    background-color: #6F64F8;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    color: white;
+    width: 65px;
+}
+
+.card-rating__price {
+    font-size: 1.1em;
+}
+
+.card__bottom{
+    display: flex;
+    flex-direction: row;
+}
+
+.card__description{
+    padding-top: 16px;
+}
+
+.card-description__title{
+    font-size: 1em;
+    line-height: 19px;
+    margin-bottom: 3px;
+}
+
+.card-description__desc{
+    font-size: 0.8em;
+    line-height: 14px;
+    font-weight: 300;
+    margin-bottom: 22px;
+}
+
+.card-bottom__wishlist, .card-bottom__to-cart{
+    font-weight: 600;
+    line-height: 18px;
+    font-size: 0.9em;
+    flex-basis: 50%;
+    display: flex;
+    justify-content: center;
+    padding: 14px 0;
+}
+
+.card-bottom__wishlist{
+    background: #EDEDED;
```

Comment:
please consider that potentially clickable stuff (links, buttons etc.) may have `cursor: pointer` for better experience (despite the page is not fully functional yet)